docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -118,6 +128,17 @@ The minimal age of a query in the history before it is expired. An expired
 query is removed from the query history buffer and no longer available in
 the :doc:`/admin/web-interface`.
 
+``query.remote-task.max-error-duration``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Default value:** ``5m``
+
+The amount of time that a remote task is allowed to execute while failing to
+communicate back to the coordinator. If this limit is exceeded without the
+remote task's worker node being able to reach the coordinator, the task is
```

Comment:
reword "remote task's worker node" is weird

Reply:
Not sure how to reword and still be technically accurate, not even sure if the current wording is accurate. My guess here is that the timeout is hit when a worker that's processing a task is unable to contact the coordinator with updates about that task for the specified amount of time, hence why it's worded the way it is.

Reply:
> a task is unable to contact the coordinator with updates about that task for the specified amount of time, hence why it's worded the way it is.

It's almost accurate. The direction of communication is the other way though. The coordinator pings a task every once in a while to update runtime information about a task. If coordinator fails to communicate with a task for more than the value specified by `query.remote-task.max-error-duration` it will consider task as failed.

Reply:
Reworded